core/tests/puppeteer-acceptance-tests/utilities/user/exploration-editor.ts

```diff
@@ -960,6 +960,25 @@ export class ExplorationEditor extends BaseUser {
       throw new Error(`User ${username} is not a subscriber.`);
     }
   }
+
+  async playExploration(explorationId): Promise<void> {
+    await this.page.goto(`${baseURL}/explore/${explorationId}`);
+    await this.page.waitForNavigation({waitUntil: ['load']});
```

Comment:
Combine waitForNavigation with calls for any functions that navigates. Example:
`await Promise.all([this.page.waitForNavigation(), this.clickOn(button)]);`

Reply:
done.